src/renderers/common/Renderer.js

```diff
@@ -457,7 +457,7 @@ class Renderer {
 		const { currentColorSpace } = this;
```

Comment:
`currentColorSpace` already does that, I was thinking of renaming it to `targetColorSpace`.